### PR TITLE
feat: Optimus spec based siren alerts

### PR DESCRIPTION
**Pull request by Mryashbhardwaj**
Proton PR: https://github.com/goto/proton/pull/137

### CODE REVIEW

File: ext/scheduler/airflow/__lib.py [hunk 1]
```diff
@@ -400,7 +400,7 @@ def get_run_type(context):
 def job_success_event(context):
     try:
         meta = {
-            "event_type": "TYPE_JOB_SUCCESS",
```
Comment (ahmadnaufal) [inline ext/scheduler/airflow/__lib.py:403]: is this change expected?

Comment (Mryashbhardwaj) [reply ext/scheduler/airflow/__lib.py:403]: yes this is needed

File: core/job/spec.go [hunk 2]
```diff
@@ -560,14 +562,31 @@ type WebhookSpec struct {
 	Endpoints []WebhookEndPoint
 }
 
-func NewAlertSpec(on string, channels []string, config Config) (*AlertSpec, error) {
+const (
+	DefaultSeverity  = "INFO"
```
Comment (ahmadnaufal) [inline core/job/spec.go:566]: in here default severity is INFO, while in https://github.com/goto/optimus/pull/299/files#diff-ae8202ecdfcffa6355ad138f9caf0e43b8c5163a4a831e28be335bbcc3a66f2cR28 it is warning. Both should be warning, no?

**Comment by ahmadnaufal** [comment]
other than the mentioned differences, PR looks good to me

File: core/job/spec.go (hunk 2 above)
Comment (Mryashbhardwaj) [reply core/job/spec.go:566]: yes correct, 
fixing this

File: ext/scheduler/airflow/__lib.py (hunk 1 above)
Comment (Mryashbhardwaj) [reply ext/scheduler/airflow/__lib.py:403]: No, actually you are right, this should not be done

**Pull request merged**